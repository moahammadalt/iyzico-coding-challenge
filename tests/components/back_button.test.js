import React, { Component } from 'react';
import { BrowserRouter as Router, Link } from "react-router-dom";
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';
import BackButton from '../../src/components/back_button';


// Snapshots
describe('<BackButton /> Snapshot test for BackButton component', () => {

  it('Should match the BackButton snapshot', () => {

    const wrapper = shallow(
      <Router>
        <BackButton />
      </Router>
    );

    const tree = renderer.create(wrapper).toJSON();

    expect(tree).toMatchSnapshot();
  });

});