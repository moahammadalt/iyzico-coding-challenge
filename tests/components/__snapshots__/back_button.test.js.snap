// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BackButton /> Snapshot test for BackButton component Should match the BackButton snapshot 1`] = `
<div
  className="back-btn-container col-md-1"
>
  <a
    href="/"
    onClick={[Function]}
  >
    <img
      alt="sorry, not available now"
      src="test-file-stub"
    />
  </a>
</div>
`;
